src/qtiRunner/modalFeedback/inlineRenderer.js

```diff
@@ -302,7 +302,7 @@ function renderModalFeedback(feedback, loader, renderer, $container, $itemContai
             Promise.all(
                 _.map(feedback.getComposingElements(), function (elt) {
                     //render also internal elements, such as math or img
-                    return elt.postRender(data, '', renderer).pop();
+                    return elt.postRender({}, '', renderer).pop();
```

Comment:
What's the purpose of this change?

Reply:
To fix failing unit test.
Since `data` variable doesn't exist in this context, I assume this functionality never worked and that was never QAed (https://github.com/oat-sa/tao-item-runner-qti-fe/commit/57187c4e1235200a6de30b97baba5b2ffb8bcb0b change)